src/views/Player.vue

```diff
@@ -258,11 +297,37 @@ export default {
 
         // initializing the retention array with zeros
         this.retention = Array(this.player.duration).fill(0);
+        this.submitted = Array(this.ivideo_questions.length).fill(0);
+        this.skipped = Array(this.ivideo_questions.length).fill(0);
+        this.revised = Array(this.ivideo_questions.length).fill(0);
+
+        const play_buttons = document.querySelectorAll("[data-plyr='play']")
+        play_buttons[0].addEventListener(
+          'click', this.listenToPlayButtons, false
+        );
+
+        play_buttons[1].addEventListener(
+          'click', this.listenToPlayButtons, false
+        );
+
       });
 
       player.on('play', event => {
         const instance = event.detail.plyr;
         instance.fullscreen.enter()
+
+        if (this.player.currentTime < 10 && this.player.playing==true){
+          setTimeout(() => {
+            console.log(this.player.currentTime)
+            if (this.player.currentTime == 0 && this.player.playing==true) {
+              // browser unsupported -> show error page
+              this.isSeekBarMoving = false
+              this.isBrowserSupported = false
+            }
+            this.uploadJson()
+          }, 5000);
+        }
+
```

Comment:
This is the piece of code which will help us detect/handle the cases where the video does not play (stupid browser case).

We need to do this only in the first 5 seconds. If the video is playing, then we'll wait for about 5 seconds and see if the seek bar is still at time 0. If that's the case, then isSeekBarMoving is set to false and isBrowserSupported is set to false, and the user sees an error page.